packages/genesys-spark-components/src/components/stable/gux-listbox/gux-listbox.service.ts

```diff
@@ -38,60 +70,32 @@ function setActiveOption(
   }
 }
 
-function getFirstOption(list: HTMLGuxListboxElement): ListboxOptionElement {
-  let firstOption = getListOptions(list)[0];
-
-  while (firstOption && (firstOption.disabled || firstOption.filtered)) {
-    firstOption = firstOption.nextElementSibling as ListboxOptionElement;
-  }
-
-  return firstOption;
-}
-
-function getNextOption(list: HTMLGuxListboxElement): ListboxOptionElement {
```

Comment:
```
  if (hasActiveOption(list)) {
    const availableListOptions = getAvailableListOptions(list);
    const activeOption = getActiveOption(list);
    const activeOptionIndex = availableListOptions.indexOf(activeOption);

   return availableOptionsList[activeOptionIndex + 1]
  }

  return getFirstOption(list);
```

I am missing a check if you are on the last option in the above that you should add

Reply:
Done